cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -38,11 +39,15 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 
 	target := filepath.Join(ts.Dirs.Work, "installation")
 
+	dir, err := ioutil.TempDir("", "system*")
+	suite.NoError(err)
+
 	// Run installer with source-path flag (ie. install from this local path)
 	cp := ts.SpawnCmdWithOpts(
 		suite.installerExe,
 		e2e.WithArgs(target),
 		e2e.AppendEnv(constants.DisableUpdates+"=false"),
+		e2e.AppendEnv(fmt.Sprintf("%s=%s", constants.OverwriteDefaultSystemPathEnvVarName, dir)),
```

Comment:
Seeing this, I wonder if we can use something similar for the failing autostart unit tests so that they can run concurrently, but not overwrite each other. Thanks for giving me the idea :)